Forge Networking Remastered Unity/Assets/Bearded Man Studios Inc/Scripts/Networking/ArgumentParser.cs

```diff
@@ -40,7 +40,7 @@ public static class ArgumentParser
 				string arg = args[i];
 
 				string value;
-				string key = ParseArg(arg, out value) ?? floatingArgIndex++.ToString();
+				string key = ParseArgument(arg, out value) ?? floatingArgIndex++.ToString();
```

Comment:
Same question here for the `??` operator, is this availably in .Net 3.5?
Although we are planning to move the whole project to a newer version.

Reply:
Null coalescion is perfectly fine in 3.5